misc/test-stubgenc.sh

```diff
@@ -24,7 +24,8 @@ function stubgenc_test() {
     # Compare generated stubs to expected ones
     if ! git diff --exit-code "$STUBGEN_OUTPUT_FOLDER";
     then
-        EXIT=$?
+        echo "exit status $?"
```

Comment:
There is probably no need to echo that. The `$?` will always be zero in this branch, because the `!` maps all non-zero return codes to 0.

Reply:
yeah, sorry. debug statement.  thanks for catching that.